app/assets/javascripts/upload.coffee

```diff
@@ -439,8 +438,6 @@ directUpload provides an interface to upload (multiple) files to an endpoint
 ###
 @result = undefined
 @userManuscriptUpload = (fileInput) ->
-  # update helpdesk
-  $('[data-bs-toggle="popover"]').popover()
```

Comment:
Why didn't you add an "initBootstrapPopovers()" here?

Reply:
Oh, my bad, I forgot it, fixed it.